File: SiriusQuality/SiriusQuality-Penman/src/java/SiriusQuality_Penman/PotentialET_PenmanAuxiliary.java
import  java.io.*;
import  java.util.*;
import java.time.LocalDateTime;

public class PotentialET_PenmanAuxiliary
{
    private double netRadiation;
    private double solarRadiation;
    private double hslope;
    private double VPDair;
    private double conductance;
    
    public PotentialET_PenmanAuxiliary() { }
    
    public PotentialET_PenmanAuxiliary(PotentialET_PenmanAuxiliary toCopy, boolean copyAll) // copy constructor 
    {
        if (copyAll)
        {
            this.netRadiation = toCopy.getnetRadiation();
            this.solarRadiation = toCopy.getsolarRadiation();
            this.hslope = toCopy.gethslope();
            this.VPDair = toCopy.getVPDair();
            this.conductance = toCopy.getconductance();
        }
    }
    public double getnetRadiation()
    { return netRadiation; }

    public void setnetRadiation(double _netRadiation)
    { this.netRadiation= _netRadiation; } 
    
    public double getsolarRadiation()
    { return solarRadiation; }

    public void setsolarRadiation(double _solarRadiation)
    { this.solarRadiation= _solarRadiation; } 
    
    public double gethslope()
    { return hslope; }

    public void sethslope(double _hslope)
    { this.hslope= _hslope; } 
    
    public double getVPDair()
    { return VPDair; }

    public void setVPDair(double _VPDair)
    { this.VPDair= _VPDair; } 
    
    public double getconductance()
    { return conductance; }

    public void setconductance(double _conductance)
    { this.conductance= _conductance; } 
    
}

File: SiriusQuality/SiriusQuality-Penman/src/cpp/SiriusQuality_Penman/PotentialET_PenmanAuxiliary.cpp
#include "PotentialET_PenmanAuxiliary.h"
using namespace SiriusQuality_Penman;

PotentialET_PenmanAuxiliary::PotentialET_PenmanAuxiliary() {}

double PotentialET_PenmanAuxiliary::getnetRadiation() { return this->netRadiation; }
double PotentialET_PenmanAuxiliary::getsolarRadiation() { return this->solarRadiation; }
double PotentialET_PenmanAuxiliary::gethslope() { return this->hslope; }
double PotentialET_PenmanAuxiliary::getVPDair() { return this->VPDair; }
double PotentialET_PenmanAuxiliary::getconductance() { return this->conductance; }

void PotentialET_PenmanAuxiliary::setnetRadiation(double _netRadiation) { this->netRadiation = _netRadiation; }
void PotentialET_PenmanAuxiliary::setsolarRadiation(double _solarRadiation) { this->solarRadiation = _solarRadiation; }
void PotentialET_PenmanAuxiliary::sethslope(double _hslope) { this->hslope = _hslope; }
void PotentialET_PenmanAuxiliary::setVPDair(double _VPDair) { this->VPDair = _VPDair; }
void PotentialET_PenmanAuxiliary::setconductance(double _conductance) { this->conductance = _conductance; }